src/parser/jobs/nin/modules/Ninki.js

```diff
@@ -22,12 +25,17 @@ const NINKI_SPENDERS = {
 
 export default class Ninki extends Module {
 	static handle = 'ninki'
+	static title = 'Ninki Timeline'
+	static displayMode = DISPLAY_MODE.FULL
 	static dependencies = [
 		'cooldowns',
 		'suggestions',
 	]
 
 	_ninki = 0
+	_history = {
```

Comment:
Minor, but I see no reason not to just have the `_history` variable be an array instead of an object containing an array. It makes sense for RDM (since `_history` tracks both Black and White Mana), but NIN's only got the one resource.